dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -127,7 +128,9 @@ private static void generateReport(Configuration configuration, Path output, Str
       templateData.put("groupId", groupId);
       templateData.put("artifactId", artifactId);
       templateData.put("version", version);
-      templateData.put("mavenOutput", mavenOutput);
+      // TODO should probably escape reserved characters in freemarker template instead
+      templateData.put("mavenOutput",
+          mavenOutput.replaceAll("&", "&amp;").replaceAll("<", "&lt;"));
```

Comment:
Would it be worth using `HtmlEscaper` here instead?  You don't need `&gt;`?

Reply:
No, > can appear literally in XML and does not need to be escaped.
We don't want an HTML escaper here either. We need XML escaping rules. Probably need to figure out the freemarker rules: https://freemarker.apache.org/docs/dgui_misc_autoescaping.html